draft-js/index.d.ts

```diff
@@ -357,7 +357,11 @@ declare namespace Draft {
              *   - "props": Props to be passed into the React component that will be used.
              */
             interface DraftDecorator {
-                strategy: (block: ContentBlock, callback: (start: number, end: number) => void) => void;
+                strategy: (
+                    block: ContentBlock,
+                    callback: (start: number, end: number) => void,
+                    contentState?: ContentState
```

Comment:
Is this parameter really optional? http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html